certify-core/src/main/java/io/mosip/certify/core/spi/SvgTemplateService.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.core.spi;
+
+import io.mosip.certify.core.entity.SvgTemplate;
+
+import java.util.UUID;
+
+public interface SvgTemplateService {
+    SvgTemplate getSvgTemplate(UUID id);
```

Comment:
We designed the Templates to be uniquely identifiable from the DB side by UUIDs such that it becomes easier to do the simplification implementation and Integrators don't get into implementing templates with IDs such as `National_ID_NARNIA_IOS_6_PORTRAIT_v2023_final_v3` and just keep it simple and not tied to any implementation.

Much later, not tying it to any specific naming convention leaves more control for the Certify authors to implement template management for VC DM 2.0 to be able to easily manage `0 to n` SVG templates later and also leaving room for upgraded templates for newer issued VCs.

Aside, in a recent PR comment @jainhitesh9998  suggested adding a `type` column to the DB tables so that we leave room for the spec to evolve in a way that SVG/PDF/XPS/JPG templates can also be managed later on, Let us know your thoughts.

cc @Piyush7034 WDYT (what do you think?)

Reply:
Yes, there were some more changes suggested by Hitesh on the svg-template side which includes some name changes in the entity as well

Reply:
For this change corresponding changes have to be made for services, repository and entities @vharsh

Reply:
The DB table is expecting a UUID eventually, do you want this to be changed as well?

Reply:
The above change is done, the API, service & Repository, DB now expects type to be String.

Reply:
rendering-template/{id}

Reply:
Updated in the Jira